### ISSUE TITLE
What to do: `CL:17543` axioms

### ISSUE THREAD

**Issue by joeflack4** [monarch-initiative/omim#96]
## Overview
> _Nico_: What is this massive thing doing in the omim.owl file?
> ```owl
> <rdf:Description rdf:about="http://purl.obolibrary.org/obo/CL_0017543">
>         <oboInOwl:hasExactSynonym>22) SYNDROME</oboInOwl:hasExactSynonym>
>         <oboInOwl:hasExactSynonym>3KS</oboInOwl:hasExactSynonym>
>         <oboInOwl:hasExactSynonym>3M1</oboInOwl:hasExactSynonym>
>         <oboInOwl:hasExactSynonym>3M2</oboInOwl:hasExactSynonym>
>         <oboInOwl:hasExactSynonym>3M3</oboInOwl:hasExactSynonym>
> ```

Here's an example of what one looks like in `ttl`:
```ttl
[] a owl:Axiom ;
    oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/mondo#ABBREVIATION> ;
    owl:annotatedProperty oboInOwl:hasExactSynonym ;
    owl:annotatedSource CL:0017543 ;
    owl:annotatedTarget "SUSD6" .
```

I investigated, and there's the following code:
```py
    for omim_id in omim_ids:
        ...
        if len(exact_labels) > 1:
            abbr = exact_labels.pop()
            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbr)))
            axiom_id = DeterministicBNode(abbr)
            graph.add((axiom_id, RDF.type, OWL.Axiom))
            # What's CL['0017543'] - joeflack4 2021/11/11
            graph.add((axiom_id, OWL.annotatedSource, CL['0017543']))
            graph.add((axiom_id, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
            graph.add((axiom_id, OWL.annotatedTarget, Literal(abbr)))
            graph.add((axiom_id, oboInOwl.hasSynonymType, MONDONS.ABBREVIATION))
```

Based on my comment there, this may have been something Dazhi wrote, or I may have been instructed to use `CL:0017543` but never knew what it meant. I tried looking it up [in OLS](https://www.ebi.ac.uk/ols/search?q=17543&ontology=cl) / elsewhere but had no luck.

@matentzn Do you have any idea either of what `CL:0017543` is or what kind of information we were trying to add here? Something about describing the nature of the synonym?
Do you think the issue/solution here is that (a) we don't need these axioms, (b) the `OWL.annotatedTarget` is wrong?, (c) something else?

**Comment by matentzn** [comment]
I have no idea - just drop it but make sure removing all of it does not have any other consequences other than removing.

**Comment by joeflack4** [comment]
@matentzn Sounds good. I just did so and I looked at the diff. I think the diff looks different than I'd like simply because Bnodes / these axioms appear in random order. I double checked where it looked like some of the lines were being changed, e.g.

```diff
599053c455205
<             owl:someValuesFrom OMIM:266300 ] ;
---
>             owl:someValuesFrom OMIM:300847 ] ;
599059c455211
```

But in these cases, it didn't actually replace one with the other. Both of them were still present. Just looks like the order in which these lines appeared changed.

Here's the whole diff: 
[diff.txt](https://github.com/monarch-initiative/omim/files/11780063/diff.txt)
